packages/augur-ui/src/modules/trading/components/form/form.tsx

```diff
@@ -783,6 +793,127 @@ class Form extends Component<FromProps, FormState> {
               </>
             )}
           </li>
+          <li>
+            <label className={Styles.smallLabel}>
+              {ExclamationCircle}
+              <span>
+                Max cost of{' '}
+                <span>{orderEscrowdEth === '' ? '-' : orderEscrowdEth} $ </span>{' '}
+                will be escrowed
+              </span>
+            </label>
+          </li>
+          <li>
+            <TextButtonFlip
+              text="Advanced"
+              action={() => {
+                this.setState({ showAdvanced: !s.showAdvanced });
+              }}
+              pointDown={s.showAdvanced}
+            />
+          </li>
+          {s.showAdvanced && (
+            <li>
+              <SquareDropdown
+                defaultValue={'0'}
+                options={[
+                  {
+                    label: 'Good till cancelled',
+                    value: '0',
+                  },
+                  {
+                    label: 'Order expiration',
+                    value: '1',
+                  },
+                  {
+                    label: 'Fill only',
+                    value: '2',
```

Comment:
for '0', '1', '2' can you create constants to make the code more readable. 

and use a case statement instead of nested if elses